tests/study_tests/test_multi_objective.py

```diff
@@ -161,3 +162,10 @@ def test_fast_non_dominated_sort_invalid() -> None:
         _fast_non_dominated_sort(
             np.array([[1.0, 2.0], [3.0, 4.0]]), penalty=np.array([1.0, 2.0, 3.0])
         )
+
+
+def test_normalize_value() -> None:
+    assert _normalize_value(1.0, StudyDirection.MINIMIZE) == 1.0
+    assert _normalize_value(1.0, StudyDirection.MAXIMIZE) == -1.0
+    assert _normalize_value(None, StudyDirection.MINIMIZE) == float("inf")
+    assert _normalize_value(None, StudyDirection.MAXIMIZE) == float("inf")
```

Comment:
I did not fully understand this part, because shouldn't `None` be the worst value in the given direction, i.e. `-inf`, in this case?
I would like to know what bugs you expect when we return `-inf` instead of `inf` for maximization problems with `value=None`.

Reply:
As `_normalize_value(1.0, StudyDirection.MAXIMIZE)` returns `-1.0`, this function transforms the maximization problem to the minimization problem. So that this is an expected behavior.

Reply:
Oh yeah, point taken!